03-alchemical/README.md

```diff
@@ -8,4 +8,6 @@ Here, we will generate the files for running the double-decoupling method (DDM).
 The last part is estimated analytically so no simulation will be needed. I've configured the scripts to make signs  match so that the binding free energy is just the sum of all the components: 
 
 
-DG_b = DG(attach) + DG(elec_site) + DG(vdw) + DG(elec_bulk) + DG(release)
+$\Delta G_{binding} = \Delta G_{attach} + \Delta G_{elec\_site} + \Delta G_{VDW} + \Delta G_{elec\_bulk} + \Delta G_{release}$

```

Comment:
What if we try \Delta G_{elec}^{site} and \Delta G_{elec}^{bulk} instead? The Latex rendering didn't look too good with the underscore `\_`

Reply:
@jeff231li You're right, it looks funky in jupyter. Does changing it from \$ <...> \$ to \$\$ <...> \$\$ fix it visually?

Reply:
@jaketanderson it looks like `$$ <...> $$` does make it better but putting it on superscript also works. I'm okay with either one.